Assets/Xbox Live/Scripts/XboxLiveUserManager.cs

```diff
@@ -5,7 +5,7 @@
 
 public class XboxLiveUserManager : Singleton<XboxLiveUserManager>
 {
-
+    /*
     public List<XboxLiveUserInfo> AvailableXboxLiveUsers { get; private set; }
 
```

Comment:
dead?  remove if so

Reply:
Done everywhere. Will be in the next push.